src/OpenDsc.Resource.Windows.User/Schema.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Thomas Nieto - All Rights Reserved
+// You may use, distribute and modify this code under the
+// terms of the MIT license.
+
+using System.Text.Json.Serialization;
+
+namespace OpenDsc.Resource.Windows.User;
+
+public sealed class Schema
+{
+    [JsonRequired]
+    public string userName { get; set; } = string.Empty;
```

Comment:
Use pascal case for the property names. The library will convert them into camel case.